torchaudio/backend/utils.py

```diff
@@ -46,6 +46,7 @@ def set_audio_backend(backend: Optional[str]) -> None:
     elif backend == 'sox':
         module = sox_backend
     elif backend == 'sox_io':
+        warnings.warn('"sox_io" backend is currently beta. Function signatures might change.')
```

Comment:
Let's say "experimental" instead of "beta". Should go as far as marking the backend as "_sox_io"?

Reply:
What's the difference between "beta" and "experimental"?
To me "beta" sounds it's ready to use but not in the final form yet, "experimental" sounds like we are playing around with it, which does not quite fit to my feeling. I could be wrong though.

Reply:
We have not defined either one of them formally, but [torchtext](https://github.com/pytorch/text/tree/master/torchtext/experimental) used "experimental" :)

Reply:
In this case, considering the feature lifecycle `beta` sounds more appropriate. The set of core features we want to provide is completed and confirmed to work. We are not going to change that, even though we refine some details and make small changes.

>  torchtext used "experimental"

That does not sound like a solid reasoning we should be based off of. That module is totally different nature.